administrator/components/com_banners/tmpl/banner/edit.php

```diff
@@ -55,16 +59,25 @@
 		<?php echo HTMLHelper::_('uitab.endTab'); ?>
 
 		<?php echo HTMLHelper::_('uitab.addTab', 'myTab', 'otherparams', Text::_('COM_BANNERS_GROUP_LABEL_BANNER_DETAILS')); ?>
-		<?php echo $this->form->renderFieldset('otherparams'); ?>
+			<fieldset id="fieldset-otherparams" class="options-fieldset">
+				<legend><?php echo Text::_('COM_BANNERS_GROUP_LABEL_BANNER_DETAILS'); ?></legend>
+				<?php echo $this->form->renderFieldset('otherparams'); ?>
+			</fieldset>
 		<?php echo HTMLHelper::_('uitab.endTab'); ?>
 
 		<?php echo HTMLHelper::_('uitab.addTab', 'myTab', 'publishing', Text::_('JGLOBAL_FIELDSET_PUBLISHING')); ?>
 		<div class="row">
 			<div class="col-md-6">
-				<?php echo LayoutHelper::render('joomla.edit.publishingdata', $this); ?>
+				<fieldset id="fieldset-publishingdata" class="options-fieldset option-fieldset-full">
```

Comment:
I'd rather have the fieldset's coming from the xml files rather than hardcoded

Reply:
so something like ??

```
<?php // If the fieldset has a label set, display it as the legend. ?>
<?php if (isset($fieldset->label)) : ?>
	<legend>
		<?php echo Text::_($fieldset->label); ?>
	</legend>
<?php endif; ?>
<?php if (isset($fieldset->description) && trim($fieldset->description)) : ?>
	<p>
		<?php echo $this->escape(Text::_($fieldset->description)); ?>
	</p>
<?php endif; ?>
```